src/backend/aspen/api/views/groups.py

```diff
@@ -81,32 +81,31 @@ async def get_group_members(
     group_id: int,
     request: Request,
     db: AsyncSession = Depends(get_db),
-    user: User = Depends(get_auth_user),
+    authorized_query: Select = Depends(require_access("read", Group)),
 ) -> GroupMembersResponse:
-    if user.group.id != group_id and not user.system_admin:
-        raise ex.UnauthorizedException("Not authorized")
-    group_members_query = (
-        sa.select(User).where(User.group_id == group_id).order_by(User.name.asc())  # type: ignore
-    )
-    group_members_result = await db.execute(group_members_query)
-    group_members = group_members_result.scalars().all()
-    return GroupMembersResponse.parse_obj({"members": group_members})
+    group_with_members_query = authorized_query.options(  # type: ignore
+        joinedload(Group.members)
```

Comment:
Ahhh - here we want to update the query to grab data from `user_roles` and include the role info!